### PR TITLE
Fix the fields for import

### PR DESCRIPTION
**Pull request by aunroel**

### CODE REVIEW

File: data-shaping/output/output.py [hunk 1]
```diff
@@ -561,7 +561,7 @@ def transfer_trainees(path, sheets):
                 hcpc_counter_signoff_required=row['RGHCSREQ'],
                 hcpc_signoff_name=row['RGHCNM'],
                 hcpc_signoff_number=row['RGHCNUM'],
-                ahcs_registration=row['RGAHCS'],
+                ahcs_registration=row.get('RGAHCS', 0),
```
Comment (William-Blackie) [inline data-shaping/output/output.py:564]: Hey @aunroel, it would be great to have some comments here on why we're supplying a default here?

**Review by William-Blackie — COMMENTED** [review]
Hey @aunroel, 

this looks generally good, clearly some spelling mistake fixes and having checked this out and ran it, that looks good. Could you explain why we have the default added here?

File: data-shaping/output/output.py (hunk 1 above)
Comment (aunroel) [reply data-shaping/output/output.py:564]: this is related to https://github.com/torchbox/NSHCS-TMS/pull/7

attempting to access non-existent column (which is the case for `RGAHCS` results in a script error

reason for defaulting to `0` here is simple - it's a numeric field and OutSystem defaults to `0` as null identifiers under the hood anyway.

so this is a pure workaround

Comment (William-Blackie) [reply data-shaping/output/output.py:564]: Cool, makes sense.

**Review by William-Blackie — APPROVED** [review]

**Pull request merged**